packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ext .js,.vue && stylelint ./*.scss",
+    "test": "yarn lint"
```

Comment:
These should be removed from the context of this PR since ESLint hasn't been added yet.

Reply:
removed.